docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Oracle Database Connector
+
+The [Elastic Oracle Database connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/oracle.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.7_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Oracle Database details](#gather-oracle-database-details)
+2. [Configure Oracle Database connector](#configure-oracle-database-connector)
+
+#### Gather Oracle Database details
+
+Collect the information that is required to connect to your Oracle Database:
+
+- The server host address where the Oracle Database is hosted.
+- The port where the Oracle Database is hosted.
+- The username the connector will use to log in to the Oracle Database.
+- The password the connector will use to log in to the Oracle Database.
+- The database name where the connector will query data.
+
+#### Configure Oracle Database connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+##### `host`
+
+The server host address where the Oracle Database is hosted. Default value is `127.0.0.1`. Examples:
+
+  - `192.158.1.38`
+  - `demo.instance.demo-region.demo.service.com`
+
+##### `port`
+
+The port where the Oracle Database is hosted. Default value is `9090`. Note: For a secured connection, a user needs to configure a SSL port.
+
+  - `5432`
+  - `9090`
+
+##### `username`
+
+The username of the account for the Oracle Database. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the Oracle Database. Default value is `Password_123`.
+
+##### `database`
+
+Name of the Oracle database. Default value is `xe`. Examples:
```

Comment:
Should we have no default for this setting and simply document that the default database name when using Oracle Database Express Edition is `xe`?

Reply:
Added note for all default parameters [here](https://github.com/elastic/connectors-python/pull/556/files#diff-f8ff34865ce88864196f2848aebde961f27a4c1e8ce77313a4254dd948d32cfbR441)